schemas/groups/propulsion.json

```diff
@@ -64,6 +41,12 @@
       "units": "Hz"
     },
 
+    "maxRevolutions": {
```

Comment:
Good question, raises a broader point:
- The type of engine(s), transmission(s) etc could be considered a part of design, where the *state* of the engine - being more static - could be separated. We had a similar discussion about sails. 
- The other side of the coin is - just as with sails - if you put the static properties in `design` propulsion related data becomes dispersed, making it harder for clients to retrieve information. That said, maybe we ought to designate `design` specifically as a group that contains information about the design of the vessel (hull, masts, interior, etc). Stuff that was decided upon when the boat was built and isn't dynamic at all.